contracts/Vault.vy

```diff
@@ -1060,21 +1060,22 @@ def addStrategy(
     assert strategy != ZERO_ADDRESS
 
     assert msg.sender == self.governance
+    assert self.debtRatio + debtRatio <= MAX_BPS
```

Comment:
Question: so in a live scenario where we have 2 strats with ratios 50% and 50% (i guess we don't want full capacity always in practice) to be able to add a 3rd strat with a 10% budget governance needs at least 1 transaction before to set either 1 of the inital 2 strats down to 40% or 45% 45% , is my understanding correct?

Reply:
Yes